timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue

```diff
@@ -161,7 +161,7 @@ limitations under the License.
                 <strong>Last run:</strong>
               </td>
               <td style="border: none">
-                <span> {{ timeline.created_at }} UTC </span>
+                <span> {{ timeline.results[0].created_at.split(".")[0] }} UTC </span>
```

Comment:
Given that this will crash if `results` or `created_at` are empty and that you also use this in another place, it would probably make sense to make it a computed property or at least a method.

Something like (untested):

```javascript
timelineCreated: function() {
  const firstResult = this.timeline && this.timeline.results.length > 0
    && this.timeline.results[0];
  if (!firstResult) return "invalid date";
  const createdAt = firstResult.created_at && firstResult.created_at.split(".").length > 0
    && firstResult.created_at.split(".")[0];
  
  return createdAt ? createdAt : "invalid date";
}
```

If we could use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) that would be even simpler.